### ISSUE TITLE
Defning object glyphicon. Icon Color

### ISSUE THREAD

**Issue by haxpak** [MISP/misp-objects#167]
# Use Case
- User creates a new object for his case such as Device or orgnization
- User should be able to define the glyphicon that the event graph will pick up.
- Can be in the form of class attribute in the json format
- which will be parsed by the event graph generation script
'''
{ class : 'fab fa-500px' }
'''

# object color
- When the even graph object is rendered, each object can have its own color as defined by the user when creating the object
- each object definition can have the color value. 
- can be set to a predefined list too
'''
"color": {
          "type": "string"
        }
'''

**Comment by adulau** [comment]
Good idea. We would need to change a bit the standard format that is not a big deal. We can have something similar for the colour (like we do in the [taxonomy format](https://tools.ietf.org/html/draft-dulaunoy-misp-taxonomy-format-07#section-2.4.1)). For the glyphicon, we could use the same strategy as we do in the galaxy format with an icon name.
